PhysicsTools/PatAlgos/python/producersHeavyIons/heavyIonJetSetup.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import division
+
+from RecoHI.HiJetAlgos.HiRecoPFJets_cff import akCs4PFJets
+from RecoHI.HiJetAlgos.HiGenJets_cff import ak5HiGenJets
+from RecoHI.HiJetAlgos.HiGenCleaner_cff import heavyIonCleanedGenJets
+from PhysicsTools.PatAlgos.producersHeavyIons.heavyIonJets_cff import *
+from PhysicsTools.PatAlgos.tools.helpers import *
+
+def setupHeavyIonJets(process, tag, radius, task):
+    addToProcessAndTask(
+        tag + 'Jets',
+        akCs4PFJets.clone(rParam = radius / 10),
+        process,
+        task)
+
+    from PhysicsTools.PatAlgos.tools.jetTools import addJetCollection
+
+    addJetCollection(
+        process,
+        labelName = 'AKCs4PF',
+        jetSource = cms.InputTag('akCs4PFJets'),
+        btagDiscriminators = [
+            'simpleSecondaryVertexHighEffBJetTags',
+            'simpleSecondaryVertexHighPurBJetTags',
+            'combinedSecondaryVertexV2BJetTags',
+            'jetBProbabilityBJetTags',
+            'trackCountingHighEffBJetTags',
+            'trackCountingHighPurBJetTags'
+        ],
+        genJetCollection = cms.InputTag('ak4HiCleanedGenJets'), 
+        jetCorrections = ('AK4PF', ['L2Relative'], 'None'),
+        getJetMCFlavour = False, # jet flavor disabled
+    )
+    process.patJetsAKCs4PF.useLegacyJetMCFlavour = True
+    process.patJetCorrFactorsAKCs4PF.useNPV = False
+    process.patJetCorrFactorsAKCs4PF.useRho = False
+
+def setupHeavyIonGenJets(process, tag, radius, task):
+    genjetcollection = 'ak' + str(radius) + 'HiGenJets'
+
+    addToProcessAndTask(
+        genjetcollection,
+        ak5HiGenJets.clone(rParam = radius / 10),
+        process,
+        task)
+
+    addToProcessAndTask(
+        'ak' + str(radius) + 'HiCleanedGenJets',
+        heavyIonCleanedGenJets.clone(src = genjetcollection),
+        process,
+        task)
+
+    process.patJetPartonMatchAKCs4PF.matched = "cleanedPartons"
+
+
+def useCsJetsForPat(process):
+    process.patJets = process.patJetsAKCs4PF.clone()
```

Comment:
back to the earlier thread https://github.com/cms-sw/cmssw/pull/30898#discussion_r481212973
I was trying to say that `addJetCollection` is not supposed to be mixed with using the default patJets,
which is done here.
- either do not use addJetCollection and use a modifier to update the default patJets
- or use addJetCollection and do not touch the default patJets or slimmedJets

Reply:
I took the first option.  Since this is the standard jet collection for heavy-ions, I feel like it should be used for the default patJets collection.  This means I partially reverted back to the 1st commit of the PR, except rather than deleting patJets and replacing it, I modify the PSet to achieve the same effect.